project.clj

```diff
@@ -2,7 +2,7 @@
   :min-lein-version "2.5.0"
 
   :dependencies
-  [[com.firebolt/firebolt-jdbc "2.2.4"]]
+  [[com.firebolt/firebolt-jdbc "2.4.5-SNAPSHOT"]]
```

Comment:
Let's not add dependency on a snapshot version as it will be easy to miss and release this. Can we first release JDBC `2.4.5`?